app/app.py

```diff
@@ -345,8 +349,10 @@ def _trigger_jobs(
                 changed_file,
             )
             if matches:
+                ocm_env = "stage" if "stage" in changed_file else "production"
```

Comment:
`ocm_env` can be matches in the regex (line 347)

Reply:
matches = re.match(
                 r"addons/(?P<product>.*)/addonimagesets/(?P<env>production|stage)/.*.yaml",
                 changed_file,
             )
matches.group("env")

Reply:
yes, thank you, updating the code!